lib/active_fedora/file.rb

```diff
@@ -139,6 +145,20 @@ def metadata?
       false
     end
 
+    def external_content?
+      external_url.present?
+    end
+
+    def external_url
+      return nil unless mime_type.start_with? "message/external-body"
+      url = mime_type.split(';').at(2)
+      url.nil? ? nil : url[/\"(.*?)\"/, 1]
```

Comment:
Is there a HTTP header parser we can use that would be more robust against slight changes to the external-body format?  E.g., I know the spaces are optional, and I think you might be able to use single quotes or change the order of the tokens so the URL was in the second position, etc.